sktime/datatypes/_series/_convert.py

```diff
@@ -172,3 +174,67 @@ def convert_np_to_UvS_as_Series(obj: np.ndarray, store=None) -> pd.Series:
 
 
 convert_dict[("np.ndarray", "pd.Series", "Series")] = convert_np_to_UvS_as_Series
+
+
+# obtain other conversions from/to numpyflat via concatenation to numpy3D
+def _concat(fun1, fun2):
+    def concat_fun(obj, store=None):
+        obj1 = fun1(obj, store=store)
+        obj2 = fun2(obj1, store=store)
+        return obj2
+
+    return concat_fun
+
+
+def _extend_conversions(mtype, anchor_mtype, convert_dict):
+    keys = convert_dict.keys()
+    scitype = list(keys)[0][2]
+
+    for tp in set(MTYPE_LIST_SERIES).difference([mtype, anchor_mtype]):
+        if (anchor_mtype, tp, scitype) in convert_dict.keys():
+            convert_dict[(mtype, tp, scitype)] = _concat(
+                convert_dict[(mtype, anchor_mtype, scitype)],
+                convert_dict[(anchor_mtype, tp, scitype)],
+            )
+        if (tp, anchor_mtype, scitype) in convert_dict.keys():
+            convert_dict[(tp, mtype, scitype)] = _concat(
+                convert_dict[(tp, anchor_mtype, scitype)],
+                convert_dict[(anchor_mtype, mtype, scitype)],
+            )
+
+
+if _check_soft_dependencies("xarray", severity="none"):
+    import xarray as xr
+
+    def convert_xrdataarray_to_Mvs_as_Series(
+        obj: xr.DataArray, store=None
+    ) -> pd.DataFrame:
+        if not isinstance(obj, xr.DataArray):
+            raise TypeError("input must be a xr.DataArray")
+
+        if isinstance(store, dict):
+            store["coords"] = list(obj.coords.keys())
+        return obj.to_pandas()
+
+    convert_dict[
+        ("xr.DataArray", "pd.DataFrame", "Series")
+    ] = convert_xrdataarray_to_Mvs_as_Series
+
+    def convert_Mvs_to_xrdatarray_as_Series(
+        obj: pd.DataFrame, store=None
+    ) -> xr.DataArray:
+        if not isinstance(obj, pd.DataFrame):
+            raise TypeError("input must be a xr.DataArray")
+
+        result = xr.DataArray(obj.values, coords=[obj.index, obj.columns])
```

Comment:
Note, I don't use the pandas.to_xarray here since this has some strange effects on the dimension which are reversed in that case. The workaround of using transpose would change the naming of the index and the row dim. Thus, creating an xr.DataArray explicitly here is more transparent.

Reply:
odd, but as long as it works?